dataeng/resources/warehouse-transforms-ci-master-merges.sh

```diff
@@ -9,29 +9,20 @@ make requirements
 cd $WORKSPACE/warehouse-transforms
 
 COMMIT_ID=$(git rev-parse --short HEAD)
-# Only run CI on merge commits, otherwise exit
-HEAD_COMMIT=$(git rev-parse HEAD)
-LAST_MERGE_COMMIT=$(git log --merges origin/master --format='%H' --max-count=1)
-if [ $HEAD_COMMIT == $LAST_MERGE_COMMIT ]
-then
-    echo "This is one of merge commit, Run CI"
-else
-    echo "Exiting because not a merge commit"
-    exit 0
-fi 
-
```

Comment:
We don't need to check the above condition. If team members "Squash Commit" their pull request it will squash all the intermediate commits and pushes as one single commit, in other case if team members use "Merge Commit" even without squashing their intermediate commits the top most commit will me Merge commit, so we don't need to check this condition now.